internal/blocktx/store/postgresql/postgres_test.go

```diff
@@ -681,92 +679,8 @@ func TestPostgresStore_UpsertBlockTransactions_CompetingBlocks(t *testing.T) {
 	require.NoError(t, err)
 
 	// then
-	actual, err := sut.GetMinedTransactions(ctx, []*chainhash.Hash{txHash})
+	actual, err := sut.UpsertAndGetMinedTransactions(ctx, [][]byte{txHash[:]})
```

Comment:
Does this really cover all of the test cases from `TestPostgresStore_RegisterTransactions` ?

Maybe we should keep and adapt `TestPostgresStore_RegisterTransactions` to `TestPostgresStore_UpsertAndGetMinedTransactions`